mantle/cosa/build.go

```diff
@@ -70,3 +69,20 @@ func (build *Build) FindAMI(region string) (string, error) {
 	}
 	return "", fmt.Errorf("no AMI found for region %s", region)
 }
+
+func (build *Build) WriteMeta(path string, validate bool) error {
+	if validate {
+		valid, err := build.Validator()
+		if !valid {
+			return fmt.Errorf(
+				"refusing to render invalid metadata: %v",
+				err,
+			)
+		}
+	}
+	out, err := json.MarshalIndent(build, "", "    ")
+	if err != nil {
+		return err
+	}
+	return ioutil.WriteFile(path, out, 0666)
```

Comment:
Is `0666` the proper mode for the file? I'd generally expect we'd only need `0640`.

Reply:
Change to 0644. We do want world-readable.